about.html

```diff
@@ -3,24 +3,49 @@ <h2 id="rules">Rules <a href="#rules">¶</a></h2>
-<li>Bots may only interact with a user when they're invited by that user to do so.</li>
-<li>If you need to post with a high frequency, please set your posts to be <strong>unlisted</strong> to keep the local timeline readable.</li>
-<li>In discussions, please remain civil, do not insult the people you're talking to, and try to avoid irony, sarcasm, or similar modes of language – they typically don't translate well to written language and tend to escalate discussions or misunderstandings.</li>
+<li>Actions intended to damage this instance or its performance may lead to immediate account suspension.</li>
+<li>Content that is illegal in Germany will be deleted and may lead to immediate account suspension.</li>
 </ul>
 <br>
+<h2 id="best-practices">Best practices <a href="#best-practices">¶</a></h2>
+
+<p>The list below is a collection of behaviour that we expect to see from our users. If you see a user go against these best practices in a way that bothers you, please file a report and we will talk to them. While these best practices are designed to be guidelines for a good communal instance, <strong>repeated malicious unwillingness</strong> to follow the best practices will be considered just like breaking a rule.</p>
+
+<ul>
+  <li>In general, use the tools provided to foster a considerate and accessible atmosphere. This includes the liberal use of content warnings (especially on potentially disturbing or controversial topics), and alt-text captioning of media files.</li>
+  <li>When possible, provide credit for creative works in your posts that are not your own.</li>
+  <li>Uninvited comments about another user's personal choices, lifestyle or family are strongly discouraged and may be considered harassment. Inappropriate sexual attention, comments about appearance and implication of physical contact will not be tolerated toward any non-consenting user.</li>
+  <li>Posts with sexual content or gore should be tagged.</li>
+  <li>Advertisements have to be tagged, and must not be excessive or automated.</li>
+  <li>Bots may only interact with a user when they're invited by that user to do so.</li>
+  <li>Automated posts and high-frequency posts should be <strong>unlisted</strong> (rendering visible to everybody, but not appearing on the local timeline) to keep the local timeline of our instance a place of community dialogue and human interaction. This extends to bots, feed posters, Twitter "retweets" and Twitter crossposts with broken mentions. Crossposter accounts that stop being active participants in our community may be removed at the discretion of the moderators.</li>
+  <li>In discussions, please remain civil, do not insult the people you're talking to, and try to avoid irony, sarcasm, or similar modes of language – they typically don't translate well to written language and tend to escalate discussions or misunderstandings.</li>
```

Comment:
```diff
-   <li>In discussions, please remain civil, do not insult the people you're talking to, and try to avoid irony, sarcasm, or similar modes of language – they typically don't translate well to written language and tend to escalate discussions or misunderstandings.</li>
+   <li>In discussions, please remain civil, do not insult the people you're talking to. Please note that irony, sarcasm and similar modes of language typically don't translate well to written language and tend to escalate discussions or misunderstandings.</li>
```

Reply:
That's a good way of phrasing it, and I'll adopt a similar wording. Of course irony and sarcasm are allowed, but should be used cautiously and without intent to aggravate or escalate discussions. Thank you for your suggestion :)